src/domain/ecoverse/ecoverse.resolver.mutations.ts

```diff
@@ -74,6 +87,7 @@ export class EcoverseResolverMutations {
     return challenge;
   }
 
+  @Roles(RestrictedGroupNames.EcoverseAdmins)
```

Comment:
why does this not have gqlauth guard on it?

Reply:
I was sloppy at copy-pasting. Fixed (in many places) in fa8c08641f46d2ba3f5f500c4558bccc6cf14ab6